test/decoder_test.exs

```diff
@@ -0,0 +1,39 @@
+defmodule DecoderTest do
+  use ExUnit.Case
+
+  import Membrane.Testing.Pipeline.Assertions
+
+  alias Membrane.Pipeline
+
+  def make_pipeline(in_path, out_path) do
+    Membrane.Testing.Pipeline.start_link(
```

Comment:
I would flatten that a bit and move `start_link` call to the `DecodingPipeline` module.
`make_pipeline` would be redundant then (if combined with changes in `DecodingPipeline` )